docs/components/connectors/out-of-the-box-connectors/outbound/rest.md

```diff
@@ -11,7 +11,7 @@ If you use the REST Connector, ensure you do not have any instance variable name
 
 :::
 
-The **REST Connector** allows you to make a request to a REST API and use the response in the next steps of your process.
+The **REST Connector** is an inbound protocol Connector that allows you to make a request to a REST API and use the response in the next steps of your process.
```

Comment:
I _think_ the REST Connector is an outbound protocol Connector. @sbuettner can you confirm?